app/src/androidTest/java/ch/sdp/vibester/activity/ScoreBoardActivityTest.kt

```diff
@@ -48,16 +48,4 @@ class ScoreBoardActivityTest {
             )
         }
     }
-
-    @Test
```

Comment:
Why this test is removed? Does it relate to handle?

Reply:
The order of the users in the firebase is not deterministic. We have been deleting/adding users, so the user positions always change. I believe the removal of the test will not decrease any coverage of your changes